frontend/actions/playlist_actions.js

```diff
@@ -30,12 +30,12 @@ export const fetchPlaylists = () => dispatch => (
 
 export const createPlaylist = (defaultPlaylist) => dispatch => 
 {   
-    // debugger
-    return (
-    postPlaylist(defaultPlaylist)
-        .then( defaultPlaylist => dispatch( receiveCurrentPlaylist(defaultPlaylist) ),
-        err => (dispatch( receivePlaylistErrors(err.responseJSON) )))
-)};
+    return postPlaylist(defaultPlaylist)
+        .then( (respObj) => {
+            dispatch( fetchPlaylists() );
+            return respObj.playlist.id;
+        }, err => (dispatch( receivePlaylistErrors(err.responseJSON) )))
+};
```

Comment:
nit: might format this differently, you should really install a linter like [prettier](https://github.com/prettier/prettier)

Reply:
Will clarify formatting going forward, ty